infrastructure/clients/convene-test-vultr.zinc.coop/infrastructure.tf

```diff
@@ -0,0 +1,84 @@
+variable "public_key" {
+  type = string
+}
+
+variable "cloudflare_email" {
+  type = string
+}
+
+
+variable "cloudflare_api_key" {
+  type = string
+}
+
+variable "cloudflare_zone_id" {
+  type = string
+}
+
+provider "cloudflare" {
+  version = "~> 2.0"
+  email   = var.cloudflare_email
+  api_key = var.cloudflare_api_key
+}
+
+# Create a record
+resource "cloudflare_record" "convene-test-vultr" {
+  zone_id = var.cloudflare_zone_id
+  name    = "convene-test-vultr"
+  value   = vultr_server.convene_vultr_video.main_ip
+  type    = "A"
+  ttl     = 1
+}
+
+variable "vultr_api_key" {
+  type = string
+}
+
+provider "vultr" {
+  api_key = var.vultr_api_key
+}
+
+resource "vultr_server" "convene_vultr_video" {
+  snapshot_id = "b765f07738a83"
```

Comment:
Could we parameterize this to a variable? Or can we use a [vultr_snapshot](https://www.terraform.io/docs/providers/vultr/d/snapshot.html) data source to filter to the `convene-jitsi-convene-test-vultr.zinc.coop*` description we use when building the image?

Reply:
Couldn't figured out a way to do `convene-jitsi-convene-test-vultr.zinc.coop*` but parameterize Vultr Snapshot id.

Reply:
That's fine! This is great work!